src/views/toolbar/data-toolbar-directive.js

```diff
@@ -231,7 +231,33 @@ angular.module('patternfly.views').directive('pfDataToolbar',
         $scope.checkViewDisabled = function (view) {
           return $scope.config.viewsConfig.checkViewDisabled && $scope.config.viewsConfig.checkViewDisabled(view);
         };
+
+        $scope.filterExists = function (filter) {
+          var found = false;
+          $scope.config.filterConfig.appliedFilters.forEach(function (nextFilter) {
```

Comment:
Seems like we could use [`_.where`](https://lodash.com/docs#where) here to simplify this.